tests/unit/ranx/normalization/min_max_norm_test.py

```diff
@@ -43,11 +43,9 @@ def test_min_max_norm(run):
     assert norm_run["q2"]["d2"] == (2 - 1) / (2 - 1)
 
 
-def test_min_max_norm_with_invert(run):
+@pytest.mark.parametrize("norm_name", ["min_max_inverted", "min_max-inverted"])
```

Comment:
This runs two tests: first with  `"min_max_inverted"` as value for `norm_name` and then as `"min_max-inverted"` as value  for `norm_name`.

Reply:
I truly appreciate you taking the time to refine my code. Your recommendations are spot-on, and I wholeheartedly agree with the decision to keep the `invert` parameter hidden from the users. By doing so, we can provide a more streamlined and user-friendly experience when working with ranx.